futures-util/src/stream/mod.rs

```diff
@@ -1095,7 +1095,7 @@ pub trait StreamExt: Stream {
     /// let (tx, mut rx) = mpsc::channel(1);
     /// let mut total = 0;
     /// let mut tx = Some(tx);
-    /// let mut async_tasks = rx.buffer_unordered(2);
+    /// let mut async_tasks = rx.buffer_unordered(2).fuse();
```

Comment:
I'd really like to show an example like the one I have [here](https://github.com/rust-lang-nursery/futures-rs/pull/1323/files#diff-bdc0724f41a9872d82003eae02367ecbR180) where we push new tasks onto a `FuturesUnordered` after the loop has started, since I think that's a key usecase that is the primary reason (IMO) for this feature.

Reply:
Yep, I'll use that example as is, it's definitely more succinct than the channel based one.

Reply:
Thanks! :)